_data/projects/event/awkward-array.yml

```diff
@@ -1,4 +1,5 @@
 name: awkward-array
+logo: /assets/images/projlogo/logo_awk.png
```

Comment:
I would have written `projlogos`, but not a big deal at all.

Reply:
For such reasonably short package names, why not write them in full? Here and in a couple of other examples?